xtask/src/cargo.rs

```diff
@@ -312,4 +317,32 @@ impl<'a> CargoToml<'a> {
             &mut handle_dependencies,
         );
     }
+
+    pub fn package(&self) -> Package {
+        self.package
+    }
+
+    pub fn repo_dependencies(&mut self) -> Vec<Package> {
```

Comment:
A small comment explaining what this does would be beneficial I think, I ended up reading the code itself to understand exactly what it was doing.